nym-wallet/nym-wallet-types/src/network.rs

```diff
@@ -115,6 +115,7 @@ mod sandbox {
     pub(crate) const COCONUT_DKG_CONTRACT_ADDRESS: &str =
         "nymt1k8re7jwz6rnnwrktnejdwkwnncte7ek7kk6fvg";
     pub(crate) const SERVICE_PROVIDER_DIRECTORY_CONTRACT_ADDRESS: &str = "";
+    pub(crate) const NAME_SERVICE_CONTRACT_ADDRESS: &str = "";
```

Comment:
if it doesn't exist, does the entry even need to be here? won't it just be parsed into a `None` instead?

Reply:
Yep we can explicitly set it to `None`. I'll do that